src/components/DocumentControlsPanel/DocumentEditPanel/DocumentEditPanel.cypress.js

```diff
@@ -12,26 +12,28 @@ describe("DocumentEdit", function() {
 
   // https://on.cypress.io/interacting-with-elements
 
-  it("DocumentEdit Save click should update document user-friendly name", function() {
+  it("Document Edit Panel Save click should update document user-friendly name", function() {
```

Comment:
Start test descriptions with `should` so they read like a sentence

`it('should update document when save button is clicked'...`

Reply:
Also, looks like you're prefixing each name in this file with `Document Edit Panel` as a way to group tests. Instead, use `describe` to do so.

```
describe('Document Edit Panel', () => {
  it('should x', () => {
    
  });
});
```